src/test/java/org/terasology/launcher/game/TestGameStarter.java

```diff
@@ -63,11 +72,19 @@ public void testJre() throws IOException {
         assertTrue(task.getRuntimePath().startsWith(Path.of(System.getProperty("java.home"))));
     }
 
-    @Test
-    public void testBuildProcess() throws IOException {
-        GameStarter starter = newStarter();
+    static Stream<Arguments> provideJarPaths() {
+        return Stream.of(
+                Arguments.of(Path.of("libs", "Terasology.jar")),
+                Arguments.of(Path.of("Terasology-5.2.0-SNAPSHOT", "lib", "Terasology.jar"))
```

Comment:
hmm. Does this actually test anything? Because the actual jar-finding code is stubbed out in this test, right?

Reply:
well, the unit under test is the `GameStarter`, not the `Installation`... so yes, this tests that the game starter takes the path from the Installation (and does not construct it on its own, as it did previously).